File: src/main/java/rd/sb_ng_browsergames/model/dto/MinesweeperScorePersistResponse.java
package rd.sb_ng_browsergames.model.dto;

public class MinesweeperScorePersistResponse {

    private boolean successfullyPersisted;
    private long id;


    public boolean isSuccessfullyPersisted() {
        return successfullyPersisted;
    }
    public void setSuccessfullyPersisted(boolean successfullyPersisted) {
        this.successfullyPersisted = successfullyPersisted;
    }
    public long getId() {
        return id;
    }
    public void setId(long id) {
        this.id = id;
    }
    
}

File: src/app/model/minesweeper-score-persist-response.ts
export class MinesweeperScorePersistResponse {

    private successfullyPersisted: boolean;
    private id: number;

    
    constructor() {
        this.successfullyPersisted = false;
        this.id = -1;
    }


    public isSuccessfullyPersisted(): boolean {
        return this.successfullyPersisted;
    }

    public setSuccessfullyPersisted(successfullyPersisted: boolean) {
        this.successfullyPersisted = successfullyPersisted;
    }

    public getId(): number {
        return this.id;
    }

    public setId(id: number) {
        this.id = id;
    }
}
